topydo/lib/ListFormat.py

```diff
@@ -74,3 +85,146 @@ def strip_placeholder_braces(p_matchobj):
     whitespace = p_matchobj.group('whitespace') or ''
 
     return before + '%' + placeholder + after + whitespace
+
+def str_clean(p_str):
```

Comment:
This function does two things, it should be split, with their names describing what it does.